From b9a1b68b22a2fe1a6721b85545c6c33a03718343 Mon Sep 17 00:00:00 2001
Message-Id: <b9a1b68b22a2fe1a6721b85545c6c33a03718343.1423872525.git.feitong.yi@intel.com>
In-Reply-To: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
References: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Wed, 21 Jan 2015 19:51:19 +0530
Subject: [PATCH 95/95] FOR_UPSTREAM [VPG]: drm/i915: Setting
 autosuspend_delay_ms to 500ms

Default value of autosuspend delay 10000ms is very high which will drain power.
We Set autosuspend delay to 500ms to aggressively go to D0i3. This
enables system to enter to S0ix much faster resulting in power saving.
By this time all timers are cancelled and if any shrinker calls runtime resume
is triggered.

Issue: GMIN-3627
Change-Id: I2471d0e888c76681ba305d589a418c507b3d4346
Signed-off-by: Deepak S <deepak.s@intel.com>
---
 drivers/gpu/drm/i915/intel_pm.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_pm.c b/drivers/gpu/drm/i915/intel_pm.c
index cc10089..03ba75a 100644
--- a/drivers/gpu/drm/i915/intel_pm.c
+++ b/drivers/gpu/drm/i915/intel_pm.c
@@ -7721,7 +7721,7 @@ void intel_init_runtime_pm(struct drm_i915_private *dev_priv)
 		return;
 	}
 
-	pm_runtime_set_autosuspend_delay(device, 10000); /* 10s */
+	pm_runtime_set_autosuspend_delay(device, 500); /* 500ms */
 	pm_runtime_mark_last_busy(device);
 	pm_runtime_use_autosuspend(device);
 
-- 
1.7.9.5

